crossbar/controller/process.py

```diff
@@ -405,9 +401,12 @@ def _start_native_worker(self, wtype, id, options=None, details=None):
 
         self._workers[id] = worker
 
-        # create a (custom) process endpoint
+        # create a (custom) process endpoint. We define a special FD3, which is
+        # used by the WAMP connection with the worker.
         #
-        ep = WorkerProcessEndpoint(self._node._reactor, exe, args, env=worker_env, worker=worker)
+        ep = WorkerProcessEndpoint(
+            self._node._reactor, exe, args, env=worker_env, worker=worker,
+            childFDs={0: "w", 1: "r", 2: "r", 3: "r"})
 
```

Comment:
It's not. We'll just have to degrade back to the old behaviour on Windows, or not support it. I'm guessing you want the former?